docs/search/opt.md

```diff
@@ -1,129 +1,129 @@
 author: CBW2007, ChungZH, TrisolarisHD, abc1763613206, Ir1d
 
-## 前言
+## Introduction
 
-DFS（深度优先搜索）是一种常见的算法，大部分的题目都可以用 DFS 解决，但是大部分情况下，这都是骗分算法，很少会有爆搜为正解的题目。因为 DFS 的时间复杂度特别高。（没学过 DFS 的请自行补上这一课）
+DFS (Depth First Search) is a really common algorithm in  OI and most of the problems can be solved using it. However, in most cases, it can only get you some points, because the time complexity of DFS is extremely high and there are very few problems where a brute force DFS can be the most optimized solution. (If you have not studied DFS, please refer to [dfs](./dfs.md) for more details).
```

Comment:
```suggestion
DFS (Depth First Search) is a really common algorithm used in OI and most of the problems can be solved using it. However, in most cases, it can only get partial points, because the time complexity of DFS is extremely high and there are very few problems that can be solved with a brute-force DFS. (If you have not studied DFS, please refer to [dfs](./dfs.md) for more details).
```
- partial points